End of year retrospective 2024 (#342)
2i2c-org/2i2c-brochure: content/blog/2024/retrospective/index.md

---
title: "2024 impact report: new team structure, new funding, and growth in our network"
date: "2024-12-08"
authors: ["Chris Holdgraf"]
tags: [report]
categories: [organization]
featured: false
draft: false
---

2024 has been a busy year for 2i2c, with many highs and lows, a lot of impact, and significant organizational change. As the year comes to an end, I want to reflect on the work we've done in 2024, and where we aim to go in 2025.

## The main idea

> In 2024, 2i2c began an organizational transformation that I now call the "$1M to $2M budget jump". This is a point in an organization's lifecycle when your team has grown enough in size and complexity that you must change the ways that you organize. The informal ways that you worked as a small group don't suffice anymore, and you have to put more effort into aligning and coordinating everyone to ensure you have impact as a group.

Reviewer: > that I now call the "$1M to $2M budget jump"
I don't have a suggested rewording for this, it just flows weirdly with the next paragraph (feel free to ignore this feedback)

---
title: "2024 impact report: new team structure, new funding, and growth in our network"
date: "2024-12-08"
authors: ["Chris Holdgraf"]
tags: [report]
categories: [organization]
featured: false
draft: false
---

2024 has been a busy year for 2i2c, with many highs and lows, a lot of impact, and significant organizational change. As the year comes to an end, I want to reflect on the work we've done in 2024, and where we aim to go in 2025.

## The main idea

> In 2024, 2i2c reached the point in an organization's lifecycle when a team has grown enough in size and complexity that you must change the ways that you organize. The informal ways that worked as a small group don't suffice anymore, and you have to put more effort into aligning and coordinating everyone to ensure you have the same impact.

Reviewer 2: ```suggestion
In 2024, 2i2c reached the point in an organization's lifecycle when a team has grown enough in size and complexity that you must change your operational style. The informal ways that worked well as a small group don't suffice anymore, and you have to put more effort into aligning and coordinating everyone to ensure you have the same impact.
```

Small style suggestion to avoid repetition.

At an organizational level, this meant a lot of introspection and planning, a few new roles, a few departing team members, a funding crunch, a successful effort to dig out of it, and a new system of work organizing our team. We'll share more about all of this later, but here are the major implications for our team:

- **We've raised another $2.2M in funding** to support our efforts in scaling and sustaining our network of community hubs. This gives us roughly another 2 years of projected runway (with some assumptions about revenue from contracts and grants).
- **We've hired and incorporated more strategic and systems-level roles** to give our team support as it grows: a Product Lead, Delivery Lead / Chief of Staff , and a People operations manager.

> - **We've re-organized our team into separate product and business development teams**, in order to focus on _providing an excellent technical platform and a collection of services that maximizes commnunity impact_, as well as _sustaining this service for our communities_.

Reviewer 2: ```suggestion
- **We've re-organized our team into separate product and business development teams**, in order to focus on _providing an excellent technical platform and a collection of services that maximizes community impact_, as well as _sustaining this service for our communities_.
```

This has allowed us to more effectively coordinate our service enhancement and development efforts, and increases our ability to deliver improvements to our communities and to upstream projects.

As a result, our organization has a much stronger foundation to build upon as we continue to grow and refine our sustainability model in 2025. It has positioned us to more effectively deal with the challenges in reaching our next milestones for scale and impact, and gives us the tools to be more adaptive and responsive to community needs.

> Along the way, we also had a lot of impact through our collaborations with communities in our network, and in the upstream projects that we support. For more details about our impact, see the summaries below.

Reviewer 2: ```suggestion
Along the way, we also generated a lot of impact through our collaborations with communities in our network, and in the upstream projects that we support. For more details about our impact, see the summaries below.
```

## Community impact

2i2c's core mission is to support its network of communities that create and share knowledge with open infrastructure. Here are the highlights of how our community network has grown and had impact.

> First, we've grown our network of hubs and users through several new partnerships, we **grew the number of active hubs from ~75 to ~105**, and **grew the number of Monthly Active Users (MAUs) from ~8100 to ~9100**.

Reviewer 2: ```suggestion
First, we've grown our network of hubs and users through several new partnerships. We **grew the number of active hubs from ~75 to ~105**, and **grew the number of Monthly Active Users (MAUs) from ~8100 to ~9100**.
```

{{< figure
  src="images/maus.png"
  width="75%"
  caption="You can see an interactive version of these numbers in our platform usage dashboard: https://2i2c.org/kpis/cloud/"
>}}

> Beyond the numbers, we also re-focused our team on reporting on impact stories from our collaborations with community members, and have published these into a (growing) list of impact reports on our blog:

Reviewer 2: ```suggestion
Beyond the numbers, we also re-focused our team on reporting impact stories from our collaborations with community members, and have published these into a (growing) list of posts on our blog:
```

Minor style suggestion, take it or leave it :)

{{< figure
  src="images/impact-gallery.png"
  width="75%"
  caption="Our impact gallery is a new place to share stories of our impact with user research communities as well as open source communities: https://2i2c.org/category/impact/"
>}}

Here are a few community highlights from this year:

> - We served around 20 communities from Latin America and Africa for the Catalyst project partner impact: https://2i2c.org/blog/2024/catalyst-partner-highlights/

Reviewer 2: ```suggestion
- We served around 20 communities from Latin America and Africa for the Catalyst project: https://2i2c.org/blog/2024/catalyst-partner-highlights/
```

- Our community partner Openscapes were invited to the White House to discuss the importance of open science: https://openscapes.org/events/2024-09-26-openscapes-whitehouse/
- The NeuroHackademy used our infrastructure to support their annual summer school: http://2i2c.org/blog/2024/neurohackademy-summer-school-reflections/
- We enabled ephemeral and sharable interactive computing environments for workshops in the geospatial community: http://2i2c.org/blog/2024/amerigeo-workshop/

> - We ran a pilot for an HHMI-funded Spyglass project for reproducing their pre-print with a live interactive environment: - BinderHub support for publishing infrastructure: https://2i2c.org/blog/2024/hhmi-spyglass-mysql/

Reviewer 2: ```suggestion
- We ran a pilot for an HHMI-funded Spyglass project for reproducing their pre-print with a live interactive environment using BinderHub to support publishing infrastructure: https://2i2c.org/blog/2024/hhmi-spyglass-mysql/
```

- We released the **JupyterHub Fancy Profiles** project, which allows for a more flexible and modern interface to launch environments with JupyterHub: 

  https://2i2c.org/blog/2024/jupyterhub-fancy-profiles-rollout/
- We used this to allow users to **build and launch custom environments in JupyterHub** in a way that users can also share with others. Here's a community with which we've piloted this functionality:

  https://2i2c.org/blog/2024/nasa-ephemeral-hubs/
- We've added a Grafana dashboard for **resource and cost monitoring with JupyterHub** to give communities more visibility over their projected cloud costs:

  https://2i2c.org/blog/2024/aws-cost-attribution/
- We began **incorporating Jupyter Book 2.0 workflows into our community hubs** and laid a foundation for enabling our communuty networks to communicate with one another more effectively using the new MyST document engine. Here are two posts about this:

  https://2i2c.org/blog/2024/project-pythia-cookoff/

  https://2i2c.org/blog/2024/jupyter-book-2/

> - We built `frx-challenge`, a tool to help communities host data challenges with secure, automated evaluation of submissions.

Reviewer 2: ```suggestion
- We built `frx-challenges`, a tool to help communities host data challenges with secure, automated evaluation of submissions.
```

  https://github.com/2i2c-org/frx-challenge
  
  This was built in collaboration with the **HHMI Cellmsp Challenge** competition:

  https://cellmapchallenge.janelia.org/

## Looking to next year

> 2025 is going to be a critical year for 2i2c to build upon the work we began this year and achieve a more sustainable and scalable community model. Here are the main areas that will guide our work in the new year and into 2026, pulled from our [recent proposal from The Navigation Fund](../funding-navigation/):

Reviewer 2: ```suggestion
2025 is going to be a critical year for 2i2c to build upon the work we began this year and achieve a more sustainable and scalable community model. Here are the main areas that will guide our work in the new year and into 2026, pulled from our [recent proposal from The Navigation Fund](/blog/2024/funding-navigation/index.md):
```
